test/distributed/fsdp/test_fsdp_use_orig_params.py

```diff
@@ -189,7 +189,8 @@ def _check_ddp_fsdp_param_parity(self, ddp_model: DDP, fsdp_model: FSDP):
             for (n1, p1), (n2, p2) in zip(
                 ddp_model.module.named_parameters(), fsdp_model.named_parameters()
             ):
-                self.assertEqual(n1, n2)
+                # Allow for FSDP prefixes
+                self.assertEqual(n1, clean_tensor_name(n2))
```

Comment:
does this mean FSDP with use_orig_params can now have FSDP prefixes? Does this create a divergence between use_orig_params and composable FSDP?

Reply:
Yes, we are allow wrapper FSDP with `use_orig_params=True` to have FSDP prefixes, and this will create a divergence with composable `fully_shard`.

To me, this divergence makes sense as it reflects the purpose of `fully_shard`.